Makefile

```diff
@@ -1,6 +1,6 @@
 # Space-separated list of the dependencies of your project (include
 # package-lint and/or buttercup if you want makel to use these tools):
-ELPA_DEPENDENCIES=package-lint
+ELPA_DEPENDENCIES=package-lint,flycheck
```

Comment:
Multiple items should be separated by spaces, not commas. (that's why the CI is failing)

That being said - I don't think it's a good idea to add flycheck here. 

Also it's the maintainers might want to check out some Emacs-specific build tool like Eldev that can simplify things like linting, running tests, etc.

Reply:
I used `Makel` because @DamienCassou very kindly answered my naive questions about Emacs :) However, I am open to any alternative (or complementary proposal)!

Reply:
> Emacs-specific build tool like Eldev that can simplify things like linting, running tests, etc.

@bbatsov: [makel](https://github.com/DamienCassou/makel) is definitely Emacs-specific and can simplify things like linting and running tests. I'm not saying it's the only or best option though. It's just mine and I use it in all my projects. makel's README suggests alternative options if you don't like it.